astropy/modeling/blackbody.py

```diff
@@ -55,10 +55,13 @@ class BlackBody1D(Fittable1DModel):
 
     Examples
     --------
+    >>> import warnings
     >>> from astropy.modeling import models
     >>> from astropy import units as u
-    >>> bb = models.BlackBody1D()  # doctest: +IGNORE_WARNINGS
```

Comment:
I really don't understand why this failed in `numpy-dev` job. It makes no sense.